backend/domain/entities.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime as dt
+
+
+class Post:
+    def __init__(self, text, author, datetime: dt = None) -> None:
+        self.text = text
+        self.author = author
+        if datetime is None:
+            datetime = dt.today()
+        self.datetime = datetime
+
+    def get_text(self):
+        return self.text
```

Comment:
There's a better way of making getters in Python using decorators
https://www.geeksforgeeks.org/python-property-decorator-property/